python/lsst/pipe/tasks/functors.py

```diff
@@ -551,6 +551,10 @@ def from_yaml(cls, translationDefinition, **kwargs):
             for flag in translationDefinition['refFlags']:
                 funcs[cls.renameCol(flag, renameRules)] = Column(flag, dataset='ref')
 
+        if 'forcedFlags' in translationDefinition:
```

Comment:
This should preferably occur after the block involving `flags`. That way, if a flag appears twice by mistaken, then the forced_src will be the one that in the object table, assuming that's what we'd want.

Reply:
...and I do want the regular meas flags to be the default.